scan/lib/scan/test_command_generator.rb

```diff
@@ -49,7 +50,9 @@ def actions
         actions = []
         actions << :clean if config[:clean]
         actions << :build unless config[:skip_build]
-        actions << :test
+        actions << "build-for-testing" if config[:build_for_testing]
+        actions << :test unless config[:test_without_building] || config[:build_for_testing]
```

Comment:
@hjanuschka Retesting sounds like a good idea! I won't be able to do that this week, but I'm interested to hear what you find :+1: